### PR TITLE
Update `new`, `help`, output styles

### PR DESCRIPTION
**Pull request by joshuayoes**
## Please verify the following:

- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR
Resolves https://github.com/infinitered/ignite/issues/2114

### CODE REVIEW

**Comment by joshuayoes** [comment]
Here is a draft of the direction I'm going
- add ignite ascii art to the beginning
- replaced extra colors with bold emphasis to create a consistent color scheme
- added link styles to all links

I want to do some more work with the spinners, but I'm pretty happy with this so far

ignite new output

**Comment by jamonholmgren** [comment]
Looking pretty good! I miss some of the colors a bit -- a lot of CLIs use monochrome, and I like that ours is more colorful. At least the red for `ignite-cli` and `Infinite Red`.

**Comment by joshuayoes** [comment]
Add npm, yarn, pnpm brand colors to output

**Comment by joshuayoes** [comment]
Add more decoration to prompts

**Comment by joshuayoes** [comment]
Added unique colors for yarn, npm, and pnpm

Screen Shot 2022-09-07 at 11 25 18 AM
Screen Shot 2022-09-07 at 11 25 09 AM
Screen Shot 2022-09-07 at 11 24 54 AM

**Comment by joshuayoes** [comment]
Added red background and white bold text to Infinite Red, along with Infinity Emoji to make it pop

Screen Shot 2022-09-07 at 6 48 59 PM

**Comment by joshuayoes** [comment]
https://user-images.githubusercontent.com/37849890/189021962-314d32e4-c364-4add-bc77-0fb7c6236a70.mov

Figured out how to replace prompt prefixs with emojis 👀

**Comment by joshuayoes** [comment]
Boolean options now have emojis, and also display 'Yes' or 'No' instead of 'true' or 'false'

https://user-images.githubusercontent.com/37849890/189189649-9d3a20d4-f1cf-42d5-b8d8-057a405c0c8c.mov

**Comment by joshuayoes** [comment]
Added some more humor to the spinner messages

Fresh install

Screen Shot 2022-09-08 at 3 57 04 PM

Cached dependencies

Screen Shot 2022-09-08 at 3 57 18 PM

**Comment by joshuayoes** [comment]
More style changes:
- Added highlights for app name, 
- Added IR highlight to Ignite version
- Changed packager colors to bg highlights

Screen Shot 2022-09-08 at 3 59 46 PM

File: src/commands/new.ts [hunk 1]
```diff
@@ -152,8 +162,6 @@ export default {
     // #endregion
 
     // #region Project Name
-    heading("🔥 About to Ignite your new app! 🔥")
```
Comment (jamonholmgren) [inline src/commands/new.ts:155]: Could we add two blank lines before the initial question? It feels weird being smashed up against the command prompt.

```ts
p()
p()
```

File: src/commands/new.ts [hunk 2]
```diff
@@ -503,116 +544,155 @@ export default {
...
+  const { strings } = toolbox
+  const { kebabCase } = strings
+
+  type Flag = keyof typeof flags
+  type FlagEntry = [key: Flag, value: Options[Flag]]
+
+  const privateFlags: Flag[] = [
+    "b",
+    "boilerplate",
+    "coloLoco",
```
Comment (jamonholmgren) [inline src/commands/new.ts:677]: Not now, but before our full launch, let's dump any references to coloLoco in the CLI.

File: test/vanilla/ignite-new.test.ts [hunk 3]
```diff
@@ -42,8 +42,8 @@ describe("ignite new", () => {
     })
 
     it("should print expected text to std out", () => {
-      expect(result).toContain(`Using ignite-cli`)
-      expect(result).toContain(`Ignite CLI ignited ${APP_NAME}`)
```
Comment (jamonholmgren) [inline test/vanilla/ignite-new.test.ts:46]: I think this line, or something equivalent, should be present, as it'll fail if Ignite fails to fully spin up an app and print the success messages.

**Review by jamonholmgren — APPROVED** [review]
2 minor nits, but otherwise I think we're ready to merge!

File: test/vanilla/ignite-new.test.ts (hunk 3 above)
Comment (joshuayoes) [reply test/vanilla/ignite-new.test.ts:46]: Ah! I knew that it was printing std out, but for some reason it did not click that we were looking for the success message

File: src/commands/new.ts (hunk 1 above)
Comment (joshuayoes) [reply src/commands/new.ts:155]: Yeah I can add that!

Comment (jamonholmgren) [reply src/commands/new.ts:155]: I like visual space -- as you can tell :D

Comment (joshuayoes) [reply src/commands/new.ts:155]: Screen Shot 2022-09-09 at 10 32 40 AM

[feat(new): add a line break before question prompts](https://github.com/infinitered/ignite/pull/2119/commits/15a89a7c0ea541b4a66eee6d30f2d01c05497036)

File: src/commands/new.ts (hunk 2 above)
Comment (joshuayoes) [reply src/commands/new.ts]: This was pretty straightforward, so I threw it in here:

[feat(new): remove references to colo-loco](https://github.com/infinitered/ignite/pull/2119/commits/d294be6998c09167492e37ae7b9ef03408a4756b)

File: test/vanilla/ignite-new.test.ts (hunk 3 above)
Comment (joshuayoes) [reply test/vanilla/ignite-new.test.ts:46]: Added here: [feat(ignite-new): add success message test](https://github.com/infinitered/ignite/pull/2119/commits/b5a16064861a24977523b5690ad20ac8c7400886)

**Pull request merged**